ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -57,6 +57,16 @@ Menu {
 
     signal menuItemClicked(int menuIndex)
 
+    function checkNotEmpty() {
```

Comment:
I got another idea, how about taking the `Menu::contentModel` and putting a `SortFilterProxyModel` with an `expression` to check for non-empty menu items? Then you could check for the proxy model's `count`